public/js/home.js

```diff
@@ -17,14 +16,12 @@ search.addEventListener('input', () => {
   })
     .then(response => response.json())
     .then((response) => {
-      // console.log(response[0])
       list.textContent = '';
       response.map((item) => {
-          const result = document.createElement('option');
-          result.textContent = item;
-          list.appendChild(result);
-          console.log(result);
+        const result = document.createElement('option');
+        result.textContent = item;
+        list.appendChild(result);
       });
     })
-    .catch(error => return(`error  : ${error}`));
+    .catch(error => (`error  : ${error}`));
```

Comment:
I don't understand what this code does. Does it log? Or does it send a message to the user?